patterns/common/sailfish-device-capabilities.yaml.template

```diff
@@ -0,0 +1,8 @@
+Description: All available capabilities. (Template file lists all, final .yaml will reflect only what device needs)
+Name: sailfish-device-capabilities
+Requires:
+%have_vibrator- pattern:sailfish-ha-vibrator
```

Comment:
Yes you're right, I forgot that the jolla-configuration-@DEVICE@.yaml that would get modified would be coming from the .spec master - device repo with submodule :P so all good, no overhead maintenance there

will adjust, thanks!